src/fhir-tool/Program.cs

```diff
@@ -607,17 +603,23 @@ private static Questionnaire.ItemComponent CreateItemComponentV2(QuestionnaireIt
             Questionnaire.ItemComponent itemComponent = new Questionnaire.ItemComponent
             {
                 Type = itemType,
-                LinkId = item.LinkId,
-                Prefix = string.IsNullOrEmpty(item.Prefix) ? null : item.Prefix,
-                Text = string.IsNullOrEmpty(item.Text) ? null : item.Text
             };
+            if (!string.IsNullOrEmpty(item.LinkId))
+                itemComponent.LinkId = item.LinkId;
+            if (!string.IsNullOrEmpty(item.Prefix))
+                itemComponent.Prefix = item.Prefix;
+            if (!string.IsNullOrEmpty(item.Text))
+                itemComponent.Text = item.Text;
 
             if (itemType != Questionnaire.QuestionnaireItemType.Group && itemType != Questionnaire.QuestionnaireItemType.Display)
             {
-                itemComponent.Required = item.Required;
+                if(item.Required.HasValue)
+                    itemComponent.Required = item.Required;
                 itemComponent.ReadOnly = item.ReadOnly;
```

Comment:
No guard for "ReadOnly"?

Reply:
item.ReadOnly will always be either true or false, no need to guard it.